### DISCUSSION TITLE
0x80070057 "The parameter is incorrect"

### DISCUSSION THREAD

**Discussion by marioapg** [microsoft/WSL#8282] [General]
Hello, I got that message when I start the WSL Ubuntu/Debian/Kali :-(

And with this command, show me that... but I have installed ubuntu, debian and kali... and nothing happens
![wsl1](https://user-images.githubusercontent.com/30729351/162454865-d2fc9caa-080e-4b71-9d35-a422d4467360.png)

I followed the complete steps for de documentation already
https://docs.microsoft.com/en-us/windows/wsl/install

I got this info with wsl.exe --status
![wsl2](https://user-images.githubusercontent.com/30729351/162455488-7f44bcad-b9e6-47a6-adbc-151a62057476.png)

Another command message
![wsl4](https://user-images.githubusercontent.com/30729351/162459602-ce6a504f-a144-41d6-9519-92f14479923c.png)

and other
![wsl5](https://user-images.githubusercontent.com/30729351/162460275-fe94f33c-35fe-4a4c-8429-a12d1d4fa0ec.png)

my system
![wsl6](https://user-images.githubusercontent.com/30729351/162460386-eb4d9062-803c-482d-b216-78307ca87872.png)

I hope someboy can help me.
Thanks a lot!

**Comment by elsaco** [comment]
@marioapg you have WSL2 installed (by looking at the output of `wsl --status` you posted) but no Linux distros. What's the output of `wsl -l -v` command? To install a distro of your choice, run `wsl --install -d Debian` for Debian or any other available. `lxrun` is not a WSL2 executable.

**Reply by marioapg** [reply]
The commands show me this

![image](https://user-images.githubusercontent.com/30729351/163695950-7c3af55b-9311-42cb-a5b4-4add45c7b2e1.png)

**Comment by ghost** [comment]
As @elsaco mentioned. You'll need to install one first.

**Reply by marioapg** [reply]
This is the commands output x_x

![image](https://user-images.githubusercontent.com/30729351/163695974-9edc4b69-bb49-414c-b503-0144cf7f7abb.png)
